styles.css

```
@@ -0,0 +1,680 @@
+h1 {
+    font-family: 'Merriweather', serif;
+    font-size: 45px;
+    font-weight: 400;
+}
+
+h2 {
+    font-family: "Merriweather", serif;
+    font-size: 25px;
+    font-weight: 600;
+}
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* unvisited link */
+header a:link {
+    color: white;
+    text-decoration: none;
+    font-family: inherit;
+}
+/* visited link */
+header a:visited {
+    color: white;
+    text-decoration: none;
+    font-family: inherit;
+}
+
+main a:link{
+    color: #5722CD;
+    text-decoration: none;
+    font-family: inherit;
+}
+
+main a:visited{
+    color: #5722CD;
+    text-decoration: none;
+    font-family: inherit;
+}
+
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    flex-basis: 1400px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 20px;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+.header-section {
+    background: rgb(75, 73, 232);
+    background: linear-gradient(135deg, rgba(75, 73, 232, 1) 43%, rgba(87, 34, 205, 1) 95%);
+    color: white;
+
+}
+
+.header-content-text {
+    text-align: center;
+    line-height: 65px;
+    padding: 90px 50px;
+}
+
+.header-content-text p {
+    font-size: 20px;
+    font-weight: 300;
+}
+
+
+.header-navigation {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+
+}
+
+.header-navigation ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 300px;
+    align-items: center;
+    color: white;
+
+}
+
+.header-content-text button {
+    background-color: #ED4385;
+    color: white;
+    border: none;
+    padding: 17px 39px;
+    font-family: inherit;
+    text-transform: uppercase;
+    font-weight: 300;
+    font-size: 13px;
+    border-radius: 40px;
+    box-shadow: 5px 5px 3px rgba(0, 0, 0, 0.15);
+
+}
+
+.header-content-image {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding: 50px 0 0 0;
+}
+
+.header-image {
+    width: 70px;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.hello-intro {
+    text-align: center;
+    line-height: 50px;
+    padding: 90px 50px;
+    color: #5722CD;
+    background-color: white;
+}
+
+
+.question-line {
+    display: flex;
+    flex-direction: row;
+    padding: 70px 20px 0 20px;
+    color: black;
+    font-weight: 600;
+    font-size: 14px;
+}
+
+/* --------------------- 4.3 Projects -----------------------*/
+.work-section {
+    background-color: #F3F6F9;
+    display: flex;
+    flex-direction: row;
```

Volgens mij moet dit
```suggestion
    flex-direction: column;
``` zijn, omdat ik bij m'n widescreen de secties naast elkaar zie staan i.p.v. onder elkaar